src/developer-quickstart.md

```diff
@@ -428,42 +428,44 @@ You should now have your outer folder, `fuel-project`, with two folders inside:
 
 ![project folder structure](./images/quickstart-folder-structure.png)
 
-#### Install the `fuels` SDK dependencies
+#### Install the `fuels` SDK dependency
 
-On this step we need to install 3 dependencies for the project:
+Fuel umbrella package includes all the main tools; `Wallet`, `Contracts`, `Providers` and more.
 
-1. `fuels`: The umbrella package that includes all the main tools; `Wallet`, `Contracts`, `Providers` and more.
-2. `fuelchain`: Fuelchain is the ABI TypeScript generator.
-3. `typechain-target-fuels`: The Fuelchain Target is the specific interpreter for the [ABI spec](https://fuellabs.github.io/fuel-specs/master/protocol/abi/index.html).
+Also, it contains the routines for ABI TypeScript generation.
 
 > ABI stands for Application Binary Interface. ABI's inform the application the interface to interact with the VM, in other words, they provide info to the APP such as what methods a contract has, what params, types it expects, etc...
 
-##### Install
+##### Installing
 
-Move into the `frontend` folder, then install the dependencies:
+Move into the `frontend` folder, then run:
 
 ```console
 $ cd frontend
-$ npm install fuels@0.24.2 --save
-added 65 packages, and audited 1493 packages in 4s
-$ npm install fuelchain@0.24.2 typechain-target-fuels@0.24.2 --save-dev
-added 33 packages, and audited 1526 packages in 2s
+$ npm install fuels --save
```

Comment:
Setting the version here will help prevent version mismatch issues. Can you elaborate more why you think it would be better to leave the version out?

Reply:
My concern is that we'd need to update this with each new SDK release, which doesn't seem realistic, IMO.

Because of this, I'm afraid this version will be constantly outdated, instructing users to use old versions.

Reply:
Maybe @luizstacio can chime in here since he was the one who initially suggested versioning when we were running into issues. WDYT,  Luiz?

Reply:
@luizstacio Anything to add here?

Reply:
Adding a version ensures that the quickstart works if we add break changes on the SDK.
Note: This quickstart focus on beta-2 and if we let this open without versioning and we need to change things on fuels or typegen to beta-3 it could break the quickstart.

Reply:
The trade-off here seems tricky.

I added the version back to the `npm install` command.